descriptor/wpkh.go

```diff
@@ -0,0 +1,181 @@
+package descriptor
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"github.com/btcsuite/btcutil/hdkeychain"
+
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcutil"
```

Comment:
shouldn't be `github.com/btcsuite/btcd/btcutil` now?

Reply:
we use github.com/btcsuite/btcutil in other pkg's, maybe we can treat this in another issue?